backend/ohq/models.py

```diff
@@ -411,3 +411,24 @@ class Announcement(models.Model):
     author = models.ForeignKey(User, related_name="announcements", on_delete=models.CASCADE)
     time_updated = models.DateTimeField(auto_now=True)
     course = models.ForeignKey(Course, related_name="announcements", on_delete=models.CASCADE)
+
+
+class VectorDB(models.Model):
```

Comment:
You can also add a `__str__` method to this model to format the name of the VectorDB objects within django admin. So perhaps:

```
def __str__(self):
      return f"{self.course}: {self.name} Vector DB"
```